VERSION

```diff
@@ -0,0 +1 @@
+0.2.2
```

Comment:
Will this be bumped?

Reply:
Current release is 0.2.2 (0.2.3 tag was pushed in error). With 0.3.0 we will bump it yes.